test/Common.FunctionalTests/AppHostConfig/HostableWebCore.config

```diff
@@ -177,8 +177,6 @@
 
     -->
     <globalModules>
-      <add name="UriCacheModule" image="%windir%\System32\inetsrv\cachuri.dll" />
```

Comment:
I feel like we should restore all modules removed here to be the same as IIS.config and IISExpress.config.

Reply:
Why?

Reply:
Not having modules makes loading unloading a bit faster.

Reply:
The original list of modules caused AVs. My only concern is that another AV is caused by this current module list which causes a red herring